actionpack/test/dispatch/content_security_policy_test.rb

```diff
@@ -211,6 +211,21 @@ def test_other_directives
     @policy.require_sri_for
     assert_no_match %r{require-sri-for}, @policy.build
 
+    @policy.require_trusted_types_for 'script'
+    assert_match %r{require-trusted-types-for script}, @policy.build
+
+    @policy.require_trusted_types_for
+    assert_no_match %r{require-trusted-types-for}, @policy.build
+
+    @policy.trusted_types "none"
+    assert_match %r{trusted-types none}, @policy.build
+
+    @policy.trusted_types "foo bar 'allow-duplicates'"
+    assert_match %r{trusted-types foo bar 'allow-duplicates'}, @policy.build
```

Comment:
Can you change this so you can pass multiple arguments and a symbol for `'allow-duplicates'` like you've done for `'script'`, e.g:

```
@policy.trusted_types "foo", "bar", :allow_duplicates
```

Thanks